### PR TITLE
Remove hidden overflow on body

### PR DESCRIPTION
**Pull request by hansmagnus**
Overflow hidden on body prevented the page to scroll when using Impress

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,6 +2,8 @@
 /js/impress.min.js
 /node_modules
 /npm-debug.log
+/DS_Store
+/examples/DS_Store
 /*.tgz
```
Comment (henrikingo) [inline .gitignore:7]: Please submit these in an independent PR.

Also, if you just add a single

    DS_Store

That will cover all possible files called DS_Store. (I think?)

File: js/impress.js [hunk 2]
```diff
@@ -364,12 +364,12 @@
...
+            // document.documentElement.style.height = "100%";
 
-            css( body, {
-                height: "100%",
-                overflow: "hidden"
-            } );
+            // css( body, {
+            //     height: "100%",
+            //     overflow: "scroll"
+            // } );
```
Comment (henrikingo) [inline js/impress.js:372]: Can you explain what you are trying to accomplish here, and why this fixes it?

**Review by henrikingo — CHANGES_REQUESTED** [review]
Hi Hans. Thanks a lot for your PR and welcome to impress.js project!

I have a comment and a questions.

Apologies for the wait. I travel a lot for work nowadays so I review PRs when I have time. But I do appreciate every one, even if I may be absent for weeks.

File: js/impress.js (hunk 2 above)
Comment (hansmagnus) [reply js/impress.js:372]: Hello, please delete this PR, I’m so sorry! Wasn’t aware I even did it.
I’m using Impress as part of the page in an LMS, so I needed the scroll to work, but the PR should never been made here like this (I probably used the wrong repository in a hurry)
Sorry again!

**Comment by hansmagnus** [comment]
This was not supposed to be PRed. Sorry again.
